exporter/collector/logs.go

```diff
@@ -163,10 +163,10 @@ func NewGoogleCloudLogsExporter(
 	cfg Config,
 	log *zap.Logger,
 	meterProvider metric.MeterProvider,
-	version string,
+	buildInfo component.BuildInfo,
```

Comment:
if we are going to make a breaking API change, should we switch to accepting exporter.Settings?  That way we can remove the logger argument, the meterprovider argument, and the version string argument.  Same comment for other new() functions

Reply:
Also note that we will need to coordinate the release of this repo with a dependency update in opentelemetry-collector-contrib

Reply:
I have the coordinated change queued up locally and ready to go once there is a new tag to refer to